deepchem/feat/base_classes.py

```diff
@@ -207,6 +208,194 @@ def __call__(self, molecules):
     return self.featurize(molecules)
 
 
+class StructureFeaturizer(Featurizer):
+  """
+  Abstract class for calculating a set of features for an
+  inorganic crystal structure.
+
+  The defining feature of a `StructureFeaturizer` is that it
+  operates on 3D crystal structures with periodic boundary conditions. 
+  Inorganic crystal structures are represented by Pymatgen structure
+  objects. Featurizers for inorganic crystal structures that are subclasses of
+  this class should plan to process input which comes as pymatgen
+  structure objects. 
+
+  Child classes need to implement the _featurize method for
+  calculating features for a single crystal.
+
+  Notes
+  -----
+  Some subclasses of this class will require pymatgen and matminer to be
```

Comment:
Might be good to add a note that since this is an abstract superclass, it shouldn't be directly instantiated

Reply:
Sounds good, I'll add a note!